util/settings/settings.go

```diff
@@ -1152,19 +1111,19 @@ func (mgr *SettingsManager) GetHelmSettings() (*v1alpha1.HelmOptions, error) {
 	return helmOptions, nil
 }
 
-// GetKustomizeSettings loads the kustomize settings from argocd-cm ConfigMap
-func (mgr *SettingsManager) GetKustomizeSettings() (*KustomizeSettings, error) {
```

Comment:
I renamed this since we're switching from the internal `KustomizeSettings` struct to the shared `v1alpha1.KustomizeOptions` struct. I opted not to change the `KustomizeOptions` struct name since it already existed.